rfc/src/rfcs/0013-list.md

```diff
@@ -141,9 +150,90 @@ If there is no result for a given field (e.g., there are no contracts), Kani wil
 
 ## Software Design
 
-We will add a new subcommand to `kani-driver`.
+### Metdata Changes
+We introduce a new `ContractedFunction` struct to `kani_metadata`:
+```rust
+pub struct ContractedFunction {
+    /// The fully qualified name the user gave to the function (i.e. includes the module path).
+    pub function: String,
```

Comment:
This is per function definition not instantiation, right?

Reply:
Yes.